bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/dto/PersistenceItemInfoBean.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.eclipse.smarthome.core.persistence.dto;
+
+/**
+ * This is a java bean that is used to serialize information about items in a persistence service to JSON.
+ *
+ * @author Chris Jackson - Initial Contribution
+ *
+ */
+public class PersistenceItemInfoBean {
```

Comment:
No - it shouldn't be used any more. I thought I'd deleted it but maybe I didn't commit it. Anyway, it's not needed. 

Sent from my iPhone

> On 2 Aug 2016, at 09:35, Kai Kreuzer <notifications@github.com> wrote:
> 
> In bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/dto/PersistenceItemInfoBean.java:
> 
> > +/**
> > + * Copyright (c) 2014-2016 by the respective copyright holders.
> > + * All rights reserved. This program and the accompanying materials
> > + * are made available under the terms of the Eclipse Public License v1.0
> > + * which accompanies this distribution, and is available at
> > + * http://www.eclipse.org/legal/epl-v10.html
> > + */
> > +package org.eclipse.smarthome.core.persistence.dto;
> > +
> > +/**
> > + * This is a java bean that is used to serialize information about items in a persistence service to JSON.
> > + *
> > + * @author Chris Jackson - Initial Contribution
> > + *
> > + */
> > +public class PersistenceItemInfoBean {
> Call it DTO, not Bean. So did you come to the conclusion that you definitely need it?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>